docker/Dockerfile.geth

```diff
@@ -0,0 +1,47 @@
+FROM alpine:3.21.2 AS build
+WORKDIR /volume
+
+RUN \
+# Install build dependencies
+    apk upgrade --no-cache \
+    && apk add --no-cache git make \
+# Clean up
+    && rm -rf /tmp/* \
+# Clone geth sources
+    && git clone https://github.com/ethereum/go-ethereum . \
+    && git checkout v1.14.5
+
+# Install go
+COPY --from=golang:1.22-alpine /usr/local/go/ /usr/local/go/
+
+# Add go to path
+ENV PATH="/usr/local/go/bin:${PATH}"
+
+# Build geth
+RUN make geth
+
+# Switch to clean container
+FROM alpine:3.21.2
+WORKDIR /volume
+
+# Copy built binary from build image
+COPY --from=build /volume/build/bin/geth /volume/geth
+
+# Set entrypoint to geth binary
+ENTRYPOINT ["/volume/geth",
```

Comment:
While testing this, I noticed each line in the entrypoint needs a terminating character `\`
```
ENTRYPOINT ["/volume/geth", \
    "--dev", \
```